README.md

```diff
@@ -1,4 +1,255 @@
+- Expansion of persistent volumes is not yet functional.
+
+## Prerequisites
+
+- Kubernetes 1.8 or higher
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+## Install the Operator
+
+<!-- Clone this repo:
+
+```
+git clone https://github.com/cockroachdb/cockroach-operator.git
+```
+
+From the `cockroach-operator` directory on your machine, create a Kubernetes cluster on GKE, specifying a cluster name (e.g., `crdb-test`).
+
+```
+./hack/create-gke-cluster.sh -c <cluster-name>
+```
+
+This creates a Kubernetes cluster on GKE, using your default GCP region and project.
+
+Install the Operator, specifying the cluster name:
+
+```
+./hack/apply-operator.sh -c <cluster-name>
+``` -->
+
+Validate that the Operator is running:
+
+```
+kubectl get pods
+
+NAME                                  READY   STATUS    RESTARTS   AGE
+cockroach-operator-6f7b86ffc4-9ppkv   1/1     Running   0          54s
+```
+
+## Start CockroachDB
+
+Download [`example.yaml`](https://github.com/cockroachdb/cockroach-operator/examples/example.yaml) and optionally modify this configuration as below.
+
+```
+curl -OOOOOOOOO https://raw.githubusercontent.com/cockroachdb/cockroach-operator/master/examples/example.yaml
+```
+
+By default `example.yaml` tells the Operator to create a secure cluster. For local testing, you can optionally deploy an insecure cluster. Exclude the following line from the configuration and skip the [Certificate signing](#certificate-signing) section below.
+
+```
+  tlsEnabled: true
+```
+
+An insecure cluster is suitable for *testing purposes only* and should not be used on production.
+
+### Certificate signing
+
+By default, the Operator uses the built-in Kubernetes CA to generate and approve 1 client and 1 node certificate for the cluster.
+
+> This differs from the way CockroachDB handles [node authentication](https://www.cockroachlabs.com/docs/stable/authentication.html#using-digital-certificates-with-cockroachdb), in which each CockroachDB node has its own node certificate. Using the Operator, one node certificate is currently used to authenticate the cluster.
+
+<!-- If you wish to use a non-Kubernetes CA, additionally enable the following lines in `example.yaml` and fill in the names of the corresponding node and client secrets:
+
+```
+nodeTLSSecret: <node-secret>
+clientTLSSecret: <client-secret>
+```
+
+For example, if you are using [`cockroach cert`](https://www.cockroachlabs.com/docs/stable/cockroach-cert.html) to generate certificates, use these secret names:
+
+```
+nodeTLSSecret: cockroachdb.node
+clientTLSSecret: cockroachdb.client.root
+``` -->
+
+### Set resource requests and limits
+
+Before deploying, we recommend explicitly allocating CPU and memory to CockroachDB in the Kubernetes pods. This is done in the `resources.requests` object in `example.yaml`. For example:
+
+```
+resources:
+  requests:
+    cpu: "16"
+    memory: "8Gi"
+  limits:
+    memory: "8Gi"
+```
+
+### Create the StatefulSet
```

Comment:
You do not create a StatefulSet you create a CR which is based on our CRD.  You can create a database.  The operator creates the StatefulSet not the user.